src/chain/section_info.rs

```diff
@@ -142,6 +142,15 @@ impl SectionInfo {
         NetworkEvent::SectionInfo(self)
     }
 
+    #[cfg(test)]
+    pub fn new_for_test(
```

Comment:
the name of the test is bit general. Also, better have a comment explains what's the purpose of the test?

Reply:
This is not a test, this is a constructor function that should only be used by tests, so that we can construct it as needed.